packages/preview-button/admin/src/translations/nl.json

```diff
@@ -0,0 +1,9 @@
+{
+  "button.preview": "Bekijken",
+  "button.cancelPreview": "Sluiten",
+  "button.copy": "Kopiëren",
+  "button.copied": "Gekopieerd",
+  "alert.htmlContentCopiedSuccess": "HTML-inhoud gekopieerd naar klembord! Plak in Word.",
```

Comment:
"Plak in Word" voelt een beetje kortaf. Misschien kunnen we het algemener en vriendelijker maken met: `De inhoud is gekopieerd naar het klembord. Je kunt de tekst nu plakken in Word.`